let User,CustomErrorHandler;_65c‍.x([["default",()=>_65c‍.o]]);_65c‍.w("../../models",[["User",["User"],function(v){User=v}]]);_65c‍.w("../../Services/CustomErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const userController = {
  async getUser(req, res, next) {
    try {
      const user = await User.findOne({ _id: req.user._id }).select(
        '-password -updatedAt -__v'
      );
      if (!user) {
        return next(CustomErrorHandler.notFound());
      }

      res.json(user);
    } catch (err) {
      return next(err);
    }
  },
};

_65c‍.d(userController);
